driver/src/main/scala/core/nodeset/authentication.scala

```diff
@@ -57,4 +60,8 @@ case class ScramSha1Authenticating(
     s"Authenticating($db, $user})"
 }
 
+case class X509Authenticating(db: String, user: String, password: String) extends Authenticating {
```

Comment:
Having an empty string instead of nothing (or at least `None`) is quite weird. Will need changes.